src/codeu/chat/client/core/UserContext.java

```diff
@@ -53,4 +54,33 @@ public ConversationContext start(String name) {
 
     return all;
   }
+
+  public void addUserInterest(String name, Uuid owner) {
```

Comment:
as with the controller, have these functions return the objects they added to the backend